php/class-photonfill-transform.php

```diff
@@ -111,14 +111,26 @@ public function transform_photon_url( $args, $image_url, $scheme = null ) {
 		 * Get the width and height of the default transform. Can be used for other transforms.
 		 */
 		public function get_dimensions( $args ) {
+			$width = ( ! empty( $args['width'] ) ) ? $args['width'] : 1000;
+			$height = ( ! empty( $args['height'] ) ) ? $args['height'] : 1000;
+
+			// An extra way to get that data.
+			if ( empty( $args['width'] ) && empty( $args['height'] ) && ! empty( $args['resize'] ) ) {
+				$dims = explode( ',', $args['resize'] );
```

Comment:
in line with my last note, this is going to lead to `$width` and `$height` being strings